_inc/client/components/product-expiration/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { moment, translate as __ } from 'i18n-calypso';
```

Comment:
If we're aiming to make this an `@automattic/component`, we might have to think about cases where i18n will be taken care of by a different module. That's why the current `@automattic/components` components provide more props, so they don't have to bundle i18n libraries and i18n functions like `translate()` or `moment()`. 


This is not a blocker for the PR, just wanted to know what your thoughts are.

Reply:
Well, I was going to import `moment` directly, as it seems Calypso is doing. I'm happy to explore how to handle this once I port this over to the components, though!

Reply:
In Calypso we try to avoid directly importing `moment` for performance reasons. It's a big library. Also, there are i18n considerations, which is why we use the `withLocalizedMoment` high order component, or the `useLocalizedMoment` hook when possible.